replica/database.hh

```diff
@@ -939,6 +942,11 @@ public:
 
     void start();
     future<> stop();
+    bool is_stopped() const noexcept {
+        return _stopped.has_value();
+    }
+    // Must be called only after the table was stopped
+    future<> wait_until_stopped() const;
```

Comment:
Should this not be private?

Reply:
Not sure why.  If stop() is public, why wouldn't wait_until_stopped() be public as well?

Reply:
Dropped in v3